app/views/courses/index.html.haml

```diff
@@ -20,6 +21,7 @@
       %td= course.start_date
       %td= course.end_date
       %td= course.allocation
+      %td= course.institute.name ? course.institute.name : "N/A" 
```

Comment:
not sure about this...

Reply:
Why have a conditional case for `name` but not the other attrs. Isn't institute `name` presence already validated?

If you're checking that the institute is valid you should probably safe navigate (`course&.institute.name`) or use `validates_associated` in the course model.